workspaces/rstudio/Dockerfile

```diff
@@ -122,35 +146,27 @@ COPY ["pl-start.sh", "pl-gosu-helper.sh", "/pl-bin/"]
 RUN mkdir -p /pl-bin && \
     chmod a+rx /pl-bin /pl-bin/pl-start.sh /pl-bin/pl-gosu-helper.sh
 
-# Warning: Do NOT try to set RStudio Server's USERID environment variable
-# here, as this will cause the home directory to be deleted and recreated (or
-# cause startup errors). The remapping to 1001:1001 is handled earlier in this
-# file.
+# Warning: In general, don't edit the variables below. Do NOT try to set
+# RStudio Server's USERID environment variable here, as this will cause the
+# home directory to be deleted and recreated (or cause startup errors). The
+# remapping to 1001:1001 is handled earlier in this file.
 
+# Set some variables for RStudio Server to ensure everything works.
 # We need to run as user rstudio (1001:1001). Do not run as root.
 USER rstudio
-# Set some variables for RStudio Server to ensure non-root mode works.
 ENV USER rstudio
+# Judging from some discussion online, the PASSWORD variable may be necessary
+# due to a bug in RStudio Server's DISABLE_AUTH mode.
```

Comment:
What bug; which discussion? If you're going to reference these things, please provide specific links for the next person who has to maintain this.

Reply:
Of all the things I bookmarked the other day, I didn't bookmark that one, so I'll have to dig it up again. It was an offhanded comment on an old Discourse thread where one of the maintainers said a fix was planned, but since it wasn't clear if that had actually happened, I added this while I was debugging the other things. In fact, I think this isn't needed, so we may not need it defensively and could take it out.